org.kde.arianna.json

```diff
@@ -0,0 +1,94 @@
+{
+    "id": "org.kde.arianna",
+    "branch": "master",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-22.08",
+    "sdk": "org.kde.Sdk",
+    "base": "io.qt.qtwebengine.BaseApp",
+    "base-version": "5.15-22.08",
+    "command": "arianna",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--filesystem=host:ro",
```

Comment:
this is a broad filesystem permission

Reply:
Yes, it's needed Arianna will search and index epubs in the home folder (using Baloo)

Reply:
home is not host.

Reply:
Ok changed

Reply:
does that mean that if baloo isn't installed on the host this won't work?

Reply:
There is a fallback for other DE, android and windows whic doesn't use baloo.